resources/web/wwi/webots.js

```diff
@@ -242,6 +244,35 @@ webots.View.prototype.setWebotsDocUrl = function(url) {
   this.webotsDocUrl = url;
 };
 
+webots.View.prototype.updateWorldList = function(currentWorld, worlds) {
+  var that = this;
+  if (worlds.length <= 1)
+    return;
+  if (typeof this.worldSelect !== 'undefined')
+    this.worldSelect.parentNode.removeChild(this.worldSelect);
+  this.worldSelect = document.createElement("select");
```

Comment:
There is no CSS for this SELECT element, right? Shall we define a nice looking CSS for it, so that it looks better integrated in the user interface?